### PR TITLE
Add Iridium-extra.app v51.1.0

### PR DESCRIPTION
**Pull request by ghost**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There was a [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates) when you submitted. It must be filled.

If you were not presented a template, you may have submitted via a CLI tool (likely [`hub`](https://hub.github.com/)). It that was the case, you may wish to see if the tool has a way to let you use the template (`hub` [is working on it](https://github.com/github/hub/issues/1118)).

In either case, please [refer to the template](https://raw.githubusercontent.com/caskroom/homebrew-cask/master/.github/PULL_REQUEST_TEMPLATE.md) and update your comment.

File: Casks/iridium-browser-extra.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask 'iridium-browser-extra' do
```
Comment (vitorgalvao) [inline Casks/iridium-browser-extra.rb]: This is wrong. Needs to be `iridium-extra`.

**Comment by vitorgalvao** [comment]
Did you just tick the boxes without running the commands? Because `audit` is failing with multiple errors.

**Comment by danwsooriginal** [comment]
Audit passes on my machine...

**Comment by adidalal** [comment]
Needs to be in *Casks/*, not *Cask/*

**Comment by danwsooriginal** [comment]
ok **now** it's working

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**